sky/clouds/service_catalog/kubernetes_catalog.py

```diff
@@ -26,3 +29,65 @@ def get_image_id_from_tag(tag: str, region: Optional[str]) -> Optional[str]:
 def is_image_tag_valid(tag: str, region: Optional[str]) -> bool:
     """Returns whether the image tag is valid."""
     return common.is_image_tag_valid_impl(_image_df, tag, region)
+
+
+def list_accelerators(
+        gpus_only: bool,
+        name_filter: Optional[str],
+        region_filter: Optional[str],
+        quantity_filter: Optional[int],
+        case_sensitive: bool = True
+) -> Dict[str, List[common.InstanceTypeInfo]]:
+    has_gpu = kubernetes_utils.detect_gpu_resource()
+    if not has_gpu:
+        return {}
+
+    label_formatter, _ = kubernetes_utils.detect_gpu_label_formatter()
+    if not label_formatter:
+        return {}
+
+    accelerators: Set[Tuple[str, int]] = set()
+    key = label_formatter.get_label_key()
+    nodes = kubernetes_utils.get_kubernetes_nodes()
+    for node in nodes:
+        if key in node.metadata.labels:
+            accelerator_name = label_formatter.get_accelerator_from_label_value(
+                node.metadata.labels.get(key))
+            accelerator_count = int(
+                node.status.allocatable.get('nvidia.com/gpu', 0))
+
+            if accelerator_name and accelerator_count > 0:
+                accelerators.add((accelerator_name, accelerator_count))
+
+    result = []
+    for accelerator_name, accelerator_count in accelerators:
+        result.append(
+            common.InstanceTypeInfo(cloud="Kubernetes",
+                                    instance_type=None,
+                                    accelerator_name=accelerator_name,
+                                    accelerator_count=accelerator_count,
+                                    cpu_count=None,
+                                    device_memory=None,
+                                    memory=None,
+                                    price=0.0,
+                                    spot_price=0.0,
+                                    region=''))
```

Comment:
Should we use region='kubernetes' here to stay consistent with the [cloud class implementation](https://github.com/skypilot-org/skypilot/blob/94b4ce31fb44f9232bce93b1febdf3c1d1291bca/sky/clouds/kubernetes.py#L49)?
```suggestion
                                    region='kubernetes'))
```